source/index.txt

```diff
@@ -22,16 +16,60 @@ The Node driver documentation is organized as follows:
   /api-documentation
   /faq
   /issues-and-help
+  /compatibility
 
-Compatibility
--------------
 
-MongoDB Compatibility
-~~~~~~~~~~~~~~~~~~~~~
+Introduction
+------------
+The official MongoDB Node.js driver allows Node.js applications to connect and perform queries with MongoDB.
+
+.. note::
+   
+   These docs are for version 3.5 of the MongoDB Node.js driver. If
+   you're looking for an older version of the MongoDB Node.js driver
+   docs, `see the GitHub pages
+   <https://mongodb.github.io/node-mongodb-native/>`_. For the main
+   MongoDB documentation, see the :manual:`MongoDB Manual </>`.
+
+Quick Start
+-----------
+Learn how to quickly get a Node.js application working with MongoDB Atlas through the `Quick Start </quick-start/>`_ section
+
+Fundamentals
+------------
+The `Fundamentals </fundamentals>`_ section defines the following core concepts involved with using the MongoDB
+Node.js driver: 
```

Comment:
Who is this section for? (Identify a persona)

Reply:
I actually disagree that the fundamentals section needs a specific persona attached to it. All users should know every concept in the fundamentals

Reply:
"All users" is a persona in this case

- Whether you're a brand new MongoDB developer _or just want to brush up on the core concepts_ then check out...

Reply:
Fixed to:
"Whether you're a brand new MongoDB developer,  or just want to brush up
on the core concepts then check out the `Fundamentals </fundamentals>`_
section that defines the following core concepts"